### DISCUSSION TITLE
Using code blocks in tabs

### DISCUSSION THREAD

**Discussion by ghost** [imfing/hextra#46] [Q&A] [answered]
[This section](https://imfing.github.io/hextra/docs/guide/shortcodes/tabs/) shows that multiple tabs can be used and the examples given are for plain text. 

I have two files `hello.py` and `hello.go` both of which print "Hello, World!". How should I use the `tabs` shortcake for these code files?

**Comment by imfing** [comment, marked as answer]
Yes, you can put Markdown there:

````

{{< tabs items="Python,Go" >}}

{{< tab >}}
```python {filename="hello.py"}
print("hello")
```
{{< /tab >}}

{{< tab >}}
```go {filename="hello.go"}
fmt.printf("Hello")
```
{{< /tab >}}

{{< /tabs >}}
````

<img width="826" alt="image" src="https://github.com/imfing/hextra/assets/5097752/441747a6-41de-4acf-baa9-201de24ddaa9">

**Reply by ghost** [reply]
Thanks !

**Reply by hugo-sid** [reply]
Hi @imfing, the screenshot which you posted above looks exciting. Would you mind sharing which application(s) is(are) in use for the Markdown and the preview?

**Reply by imfing** [reply]
@hugo-sid it's just VS Code 😄 
<img width="1230" alt="image" src="https://github.com/imfing/hextra/assets/5097752/fba268fc-35b2-4d19-9562-c92a260c0dc8">

You can also do it in your local VS Code by: `Ctrl(Cmd) + Shift + P` -> `Simple Browser: Show`, and then enter url like `http://localhost:1313`. Drag the browser tab to the right.

**Reply by hugo-sid** [reply]
Oh! it's such a simple thing 😅
I was not aware of the Simple Browser. Thanks for the info @imfing.
